Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -17,7 +17,7 @@ namespace Scenes
 {
     public abstract class Login : MonoBehaviour
     {
-        public const string MainSceneName = "SampleMain";
```

Comment:
Is there any reason for this change? This isn't a value that's likely to change during runtime so we can keep it as const

Reply:
Is LootBox scene using this script?

Reply:
No, but, based solely on the DevEx experience, I wanted to make it easy for users to change the scene they want to load, so that they don't have to step out of the editor. Every time they'd want to change the scene they'd have to go into the IDE and change this, so it's a much slower process imo.

Reply:
The reason I made it constant was because the value is unlikely to change and also it can be referenced easily however this doesn't hurt either